### PR TITLE
js if complete

### PR DESCRIPTION
**Pull request by dcatan89**

### CODE REVIEW

File: javascript-if/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (TimelikeClosure) [inline javascript-if/index.html:7]: Please `<title>` your exercises with the name of the exercise.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
Please include a screenshot(s) of your output with your pull requests.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
image

**Comment by dcatan89** [comment]
Screen Shot 2022-02-02 at 1 01 44 PM

**Comment by dcatan89** [comment]
screenshot uploaded

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**